test/svm/SvmSpoke.Bundle.ts

```diff
@@ -643,167 +656,215 @@ describe("svm_spoke.bundle", () => {
-      }).compileToV0Message([lookupTableAccount])
-    );
+      // Build the instruction to execute relayer refund leaf and write its instruction args to the data account.
+      await loadExecuteRelayerRefundLeafParams(program, owner, stateAccountData.rootBundleId, leaf, proofAsNumbers);
 
-    // Sign and submit the versioned transaction.
-    versionedTx.sign([payer]);
-    await connection.sendTransaction(versionedTx);
+      const executeInstruction = await program.methods
+        .executeRelayerRefundLeaf()
+        .accounts(staticAccounts)
+        .remainingAccounts(remainingAccounts)
+        .instruction();
+
+      // Build the instruction to increase the CU limit as the default 200k is not sufficient.
+      const computeBudgetInstruction = ComputeBudgetProgram.setComputeUnitLimit({ units: 1_400_000 });
+
+      // Create the versioned transaction
+      const versionedTx = new VersionedTransaction(
+        new TransactionMessage({
+          payerKey: owner,
+          recentBlockhash: (await connection.getLatestBlockhash()).blockhash,
+          instructions: [computeBudgetInstruction, executeInstruction],
+        }).compileToV0Message([lookupTableAccount])
+      );
 
-    // Verify all refund account balances.
-    await new Promise((resolve) => setTimeout(resolve, 1000)); // Make sure account balances have been synced.
-    const refundBalances = await Promise.all(
-      refundAccounts.map(async (account) => {
-        return (await connection.getTokenAccountBalance(account)).value.amount;
-      })
-    );
-    refundBalances.forEach((balance, i) => {
-      assertSE(balance, refundAmounts[i].toString(), `Refund account ${i} balance should match refund amount`);
+      // Sign and submit the versioned transaction.
+      versionedTx.sign([payer]);
+      await connection.sendTransaction(versionedTx);
+
+      // Verify all refund account balances (either token or claim accounts).
+      await new Promise((resolve) => setTimeout(resolve, 1000)); // Make sure account balances have been synced.
+      const refundBalances = await Promise.all(
+        refundAccounts.map(async (account, i) => {
+          if (refundType === RefundType.TokenAccounts) {
+            return (await connection.getTokenAccountBalance(account)).value.amount;
+          } else if (refundType === RefundType.ClaimAccounts) {
+            return (await program.account.claimAccount.fetch(account)).amount.toString();
+          } else if (refundType === RefundType.MixedAccounts) {
+            return i % 2 === 0
+              ? (await connection.getTokenAccountBalance(account)).value.amount
+              : (await program.account.claimAccount.fetch(account)).amount.toString();
+          }
+        })
+      );
+      refundBalances.forEach((balance, i) => {
+        assertSE(balance, refundAmounts[i].toString(), `Refund account ${i} balance should match refund amount`);
+      });
+    };
+
+    it("Execute Max Refunds to Token Accounts", async () => {
```

Comment:
this is cool. nice structure within the describe. these tests are now...complex, but that's ok.